### ISSUE TITLE
Frontend app setup

### ISSUE THREAD

**Issue by anuveyatsu** [datopian/birmingham-city-observatory#3]
## Acceptance

- [x] I have running instance of Frontend app

## Tasks

- [x] Setup a project for frontend application
- [x] Configure the domain
- [x] Setup ci/cd

**Comment by anuveyatsu** [comment]
Hi @demenech could you check Vercel and why the build is erroring please 🙏

**Comment by demenech** [comment]
@anuveyatsu 
This error was only occuring when DMS was set. It was outputting a `TypeError: Cannot read properties of undefined (reading 'map')` because there are no results for posts. I tried locally and was able to build normally without exporting the DMS setting and it gave me the same error after trying to build with this setting applied.
Fixed that. Take a look: [d8d604ae34944fdeef2acbf2546102863f7193de](https://github.com/datopian/portal-birmingham-city-observatory/commit/d8d604ae34944fdeef2acbf2546102863f7193de)
